backend/app/tests/api/routes/test_threads.py

```
@@ -133,7 +126,7 @@ def test_process_run_variants(mock_openai, remove_citation, expected_message):
 
 
 @patch("app.api.routes.threads.OpenAI")
-def test_threads_sync_endpoint_success(mock_openai, db):
+def test_threads_sync_endpoint_success(mock_openai, db, user_api_key_header):
```

Ruff (0.12.2)

Unused function argument: `db`

(ARG001)